src/state/appContext.js

```diff
@@ -0,0 +1,30 @@
+import React, { createContext, useContext, useState } from 'react';
```

Comment:
This is technically cheating because Ryan said to not use context but the rubric did say use state management. Alternatively the lab would require you pass down the artist as params all the way to the last fetch.